docs/reference/data-streams/data-streams-overview.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[data-streams-overview]]
+== Data streams overview
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+A data stream consists of one or more _backing indices_. Backing indices are
+<<index-hidden,hidden>>, automatically-generated indices used to store a
+stream's documents.
+
+image::images/data-streams/data-streams-diagram.svg[align="center"]
```

Comment:
In this image, I tend to read the graphics left to right which would imply that backing indices are numbered in descending order. I don't know if that's common, but if so, it would be better to show backing indices in increasing generation order from left to right.

Reply:
Good point. I've addressed with 9c7f0d567bf4a9ddb3bd0e86cb9d14cc75d578b2.